admin_manual/maintenance/package_upgrade.rst

```diff
@@ -2,11 +2,31 @@
 Upgrade ownCloud From Packages
 ==============================
 
-.. note:: Starting with ownCloud 8.2 the Linux package repositories have 
-   changed, and **you must configure your system to use these new 
-   repositories** to install or upgrade ownCloud 8.2+. The new repositories are 
-   at our `Open Build Service`_.
-   
+Changes in ownCloud 8.2 
+-----------------------
+
+* The Linux package repositories have changed, and **you must configure your 
+  system to use these new repositories** to install or upgrade ownCloud 8.2+. 
+  The new repositories are at our `Open Build Service`_.
+
+* Apache configuration files are in a separate package, 
+  ``owncloud-config-apache``.
+
+Upgrade Notes
+-------------
+
+If you are using a custom theme, the upgrade changes it back to the default 
+theme. After the upgrade is completed and you have verified that everything 
+works correctly, change back to your custom theme by entering the path 
+to your theme with ``'theme' => '',`` in your ``config.php`` file. If your 
+custom theme causes any issues, comment out ``'theme' => '',`` to restore the 
+default theme.
+
+``owncloud/core/skeleton/`` is overwritten on upgrades. If you wish to provide 
+a set of default files, place them in a different directory such as 
+``owncloud/config/skeleton``. Then enter this path in ``config.php`` with the 
+``'skeletondirectory' => '/path/tofile',`` directive.
```

Comment:
`path/to/folder`

It's only a naming thing, but maybe confuses the reader less.